CUDADataFormats/BeamSpot/src/classes_def.xml

```diff
@@ -0,0 +1,4 @@
+<lcgdict>
+  <class name="cms::cuda::Product<BeamSpotCUDA>" persistent="false"/>
+  <class name="edm::Wrapper<cms::cuda::Product<BeamSpotCUDA>>" persistent="false"/>
```

Comment:
do I understand correctly that all data in CUDADataFormats will be non-persistable?

Reply:
all `cms::cuda::Product` are transient-only, because:
  - they include the runtime information about the GPU context (device, CUDA stream, CUDA event for synchronisation, etc.)
  - the actual payload (`BeamSpotCUDA` in this case) is expected to live in GPU memory

The actual underlying SoA / POD could be persisted.
I don't think we had any need for this, since the end products are always legacy DataFormats, but nothing would prevent it.